frontend/src/functions/formatPhone.ts

```diff
@@ -18,6 +18,17 @@ export type PhoneString = string;
 export function formatPhone(phoneNumber: PhoneString) {
   const parsedNumber = parsePhoneNumber(phoneNumber);
 
+  // This only checks for valid length of the phone number.
+  // Ex: ('+12223333333') === true
+  if (!parsedNumber?.isPossible()) {
```

Comment:
I'm not following why we need this `if` block - if the phone number is not a valid phone number, why would be bother trying to format it? And if it's not valid, is it not possible that the input is something that would break the contents of the `if`?

Reply:
@Vinnl the function checks for length, that's it. If we know its an invalid number, we still want to format the pieces we do have. Do you have a different approach you would like us to try?

<img width="577" alt="image" src="https://user-images.githubusercontent.com/3924990/186440454-88e658c0-cf56-4387-862e-a3b43d77ec6d.png">

Reply:
Ah gotcha, thanks for the screenshot! No that makes sense, I didn't realise that we'd also show invalid phone numbers in the UI (in the error message stating that it is so - duh :sweat_smile: ).